cfy_manager/components/mgmtworker/mgmtworker.py

```diff
@@ -144,7 +155,20 @@ def _configure(self):
 
     def install(self):
         logger.notice('Installing Management Worker...')
-        self._install()
+        source_url = config[MGMTWORKER][SOURCES]['mgmtworker_source_url']
```

Comment:
this seems to just be moving it, yes, no change?

Reply:
Yup.